int-set/src/bitset.rs

```diff
@@ -104,6 +117,13 @@ impl<T> BitSet<T> {
         self.len.get()
     }
 
+    pub fn iter(&self) -> impl Iterator<Item = u32> + '_ {
```

Comment:
something I'm noticing now that I'm doing code review is that outside of the broader context of the file (e.g. just seeing this change in the diff) it's not clear that this is defined on a non-public type, which makes this look like public API. Maybe that's a reason to prefer `pub(crate)` etc even if the visibility of the containing item is non-`pub`?

Reply:
Sounds good